### PR TITLE
GPG Sign DEB and RPM packages generated by build pipeline

### PR DESCRIPTION
**Pull request by northrup**
Closes INS-1963

### CODE REVIEW

File: Builds/containers/gitlab-ci/pkgbuild.yml [hunk 1]
```diff
@@ -131,6 +132,73 @@ dpkg_build:
...
+  script:
+    - ls -alh build/rpm/packages
+    - . ./Builds/containers/gitlab-ci/sign_package.sh rpm
+
+dpkg_sign:
+  stage: sign_packages
+  dependencies:
+    - dpkg_build
+  image:
+    name: ubuntu:19.04
```
Comment (MarkusTeufelberger) [inline Builds/containers/gitlab-ci/pkgbuild.yml]: You really want/need to build with a version of Ubuntu that's going to be EOL in ~2 months (https://wiki.ubuntu.com/Releases)? Did you maybe mean 18.04 instead? Or 19.10?

Comment (seelabs) [reply Builds/containers/gitlab-ci/pkgbuild.yml]: I agree with @MarkusTeufelberger I'd rather use 18.04 here.

**Comment by MarkusTeufelberger** [comment]
Why did this get merged without review btw.?

File: Builds/containers/gitlab-ci/pkgbuild.yml (hunk 1 above)
Comment (mellery451) [reply Builds/containers/gitlab-ci/pkgbuild.yml]: we're using the container to run one command in this case (`dpkg-sig`) so I think we can make this a TODO item to switch to one of the LTS releases. In the meantime, I'm not losing any sleep over this.

**Comment by nbougalis** [comment]
@MarkusTeufelberger, it appears to have been accidental, by someone outside the C++ team at Ripple who used the GitHub workflow. We will decide what to do about this shortly.

**Comment by MarkusTeufelberger** [comment]
No biggie, I was just wondering who inside and outside Ripple (e.g. at Coil) actually has push rights here. Might be worth a few clarifying sentences in the README or a separate COONTRIBUTING file?

File: Builds/containers/gitlab-ci/pkgbuild.yml (hunk 1 above)
Comment (northrup) [reply Builds/containers/gitlab-ci/pkgbuild.yml]: I'll cut a new PR to move it to the LTS release, but as @mellery451 said I'm just using it to get _one_ package that we're running, not for actual ubuntu platform testing.

**Comment by northrup** [comment]
My apologies @MarkusTeufelberger, I was the rippler who accidentally merged without a review. My intention was to exercise the squash function on my PR so that they rolled into `develop` as one commit, however that *wasn't* what happened. Lessons learned 😞

File: Builds/containers/gitlab-ci/pkgbuild.yml (hunk 1 above)
Comment (northrup) [reply Builds/containers/gitlab-ci/pkgbuild.yml]: resolved via https://github.com/ripple/rippled/pull/3147

**Pull request merged**